e2e/ts_library/googmodule/BUILD.bazel

```diff
@@ -0,0 +1,36 @@
+load("@build_bazel_rules_nodejs//:defs.bzl", "nodejs_binary")
```

Comment:
i still don't follow the reasoning for this being in e2e rather than examples. Shouldn't we have an example of closure interop?

Reply:
Sure. I think that's a useful example to have. The one from rules_typescript was depending on a local `devmode_consumer.bzl` which seemed more like ts_library e2e to me.